config/config.exs

```diff
@@ -162,6 +162,9 @@ config :trento,
   api_key_authentication_enabled: true,
   jwt_authentication_enabled: true
 
+config :trento,
+  deregistration_debounce_seconds: :timer.seconds(0)
```

Comment:
As discussed, the default value for the deregistration debounce period should be set to 0, but remain configurable.